tracpro/static/js/poll-charts.js

```diff
@@ -91,7 +91,13 @@ jQuery.fn.extend({
 
 $(function() {
     /* Initialize date fields. */
-    $("input[class^='datepicker']").datepicker();
+    $("input[class^='datepicker']").datepicker({
+        changeMonth: true,
+        changeYear: true,
+        selectOtherMonths: true,
+        showOtherMonths: true,
+        yearRange: "2010:" + new Date().getFullYear()
```

Comment:
Any reason why we started with 2010? I'm not sure RapidPro has been around that long, looks like TextIt was introduced in 2013... so we'd most likely have no data before 2013.